process/rating/blockSigningRater_test.go

```diff
@@ -68,25 +69,66 @@ func createDefaultRatingsData() *mock.RatingsInfoMock {
 }
 
 func TestBlockSigningRater_UpdateRatingsShouldUpdateRatingWhenProposed(t *testing.T) {
-	initialRatingValue := uint32(5)
-	rd := createDefaultRatingsData()
-	shardId := uint32(0)
+	t.Parallel()
 
-	bsr, _ := rating.NewBlockSigningRater(rd)
-	computedRating := bsr.ComputeIncreaseProposer(shardId, initialRatingValue)
+	t.Run("shard before supernova", func(t *testing.T) {
+		t.Parallel()
+		initialRatingValue := uint32(5)
+		rd := createDefaultRatingsData()
+		shardId := uint32(0)
 
-	expectedValue := uint32(int32(initialRatingValue) + proposerIncreaseRatingStep)
+		bsr, _ := rating.NewBlockSigningRater(rd, &enableEpochsHandlerMock.EnableEpochsHandlerStub{})
+		computedRating := bsr.ComputeIncreaseProposer(shardId, initialRatingValue, 0)
 
-	assert.Equal(t, expectedValue, computedRating)
+		expectedValue := uint32(int32(initialRatingValue) + proposerIncreaseRatingStep)
+
+		assert.Equal(t, expectedValue, computedRating)
+	})
+	t.Run("shard after supernova", func(t *testing.T) {
+		t.Parallel()
+
+		initialRatingValue := uint32(5)
+		rd := createDefaultRatingsData()
+		shardId := uint32(0)
+
+		bsr, _ := rating.NewBlockSigningRater(rd, &enableEpochsHandlerMock.EnableEpochsHandlerStub{
+			IsFlagEnabledInEpochCalled: func(flag core.EnableEpochFlag, epoch uint32) bool {
+				return true
+			},
+		})
+		computedRating := bsr.ComputeIncreaseProposer(shardId, initialRatingValue, 0)
```

Comment:
Would've been a bit more of a complete test if we'd actually that the step is taken by calling `bsr.ratingsHandler.ShardChainRatingsStepHandler()` and for meta below